<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>368</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>6</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addhang_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>등록 추가</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="hang_cancel_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기(저장취소)</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hang_save_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>64</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>83</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_hun_hang_reg_widget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_hang_tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>기존 등록된 항</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="new_hang_tableWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>161</width>
     <height>241</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>신규등록 항</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 헌금 항  등록</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>84</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>신규등록 헌금 항</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hang_delete_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 항 삭제</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hang_delete_button_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 항 수정</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;헌금 '항'은 헌금의 대분류로서 기존 목록은 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;일반회계에 '예배', '절기헌금', '지정헌금',' 기타소득'&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;이 있으며,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;선교회계에는 '선교헌금' 이 있다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>gubun_comboBox</tabstop>
  <tabstop>new_hun_hang_reg_widget</tabstop>
  <tabstop>addhang_button</tabstop>
  <tabstop>hang_save_button</tabstop>
  <tabstop>hang_cancel_button</tabstop>
  <tabstop>hang_delete_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>hang_save_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hang_cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>filesave_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addhang_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hang_reg_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hang_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hang_delete_button_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hunhang_modify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hang_reg_input()</slot>
  <slot>hang_add_cancel()</slot>
  <slot>file_save()</slot>
  <slot>filesave_cancel()</slot>
  <slot>hunhang_modify()</slot>
  <slot>delete_selected_row()</slot>
 </slots>
</ui>
